docs/source/addons/i18n.md

```diff
@@ -31,4 +31,6 @@ In the context of your project, run `yarn i18n` to merge the add-on translations
 
 ## Override translations
 
-The workflow allows you to override translations from your project scope, so the project translations "always win".
+If you have multiple add-ons installed in your project, the translations are loaded in the order your add-ons are listed in `package.json`. If two add-ons provide different translations for the same message, then the last defined add-on wins.
```

Comment:
One more nitpick. One sentence per line, and this time I *promise* it is good to go. I apologize for my oversight.

```suggestion
If you have multiple add-ons installed in your project, the translations are loaded in the order your add-ons are listed in `package.json`.
If two add-ons provide different translations for the same message, then the last defined add-on wins.
```